cpu/cc2538/include/periph_cpu.h

```diff
@@ -112,6 +112,23 @@ typedef enum {
 } gpio_mode_t;
 /** @} */
 
+
+/**
+ * @name   UART device configuration
+ * @{
+ */
+typedef struct {
+    cc2538_uart_t *dev;       /**< pointer to the used UART device */
+    gpio_t rx_pin;            /**< pin used for RX */
+    gpio_t tx_pin;            /**< pin used for TX */
+    IRQn_Type irq;            /**< interrupt number of the UART's multifunction interrupt */
+    cc2538_ioc_pin_t rxd;     /**< function assigned to the RX pin */
```

Comment:
why make `rxd` and `txd` configurable by the board? These values are static for the RX and TX pins, so they can be hard-coded in the uart driver implementation, right?!